saleor/checkout/core.py

```diff
@@ -210,32 +212,41 @@ def billing_address(self, address):
         self.modified = True
 
     @property
-    def discount(self):
-        """Return a discount if any."""
+    def discount_amount(self):
```

Comment:
"Amount" suggests that it returns a number. `discount` was ok to keep, paired with the new `discount_name`.